playwright-e2e/tests/dsm/kitUploadFlow/kit-queue-ui.spec.ts

```diff
@@ -0,0 +1,61 @@
+import {expect} from '@playwright/test';
+import {test} from 'fixtures/dsm-fixture';
+import {WelcomePage} from 'dsm/pages/welcome-page';
+import {Navigation} from 'dsm/component/navigation/navigation';
+import {StudyEnum} from 'dsm/component/navigation/enums/selectStudyNav-enum';
+import {SamplesNavEnum} from 'dsm/component/navigation/enums/samplesNav-enum';
+import {KitsColumnsEnum} from 'dsm/pages/kitsInfo-pages/enums/kitsColumns-enum';
+import {studyShortName} from 'utils/test-utils';
+import KitQueuePage from 'dsm/pages/kitsInfo-pages/kit-queue-page';
+
+test.describe.serial('Sample Kit Queue UI', () => {
+  const studies = [
+    StudyEnum.ANGIO, StudyEnum.BASIL, StudyEnum.OSTEO, StudyEnum.OSTEO2, StudyEnum.PANCAN, StudyEnum.RGP,
```

Comment:
we might want to take basil out - I don't think we've used that in over a year

Reply:
seeing as most cmi studies are listed here, might be good to use Brain instead if we need an additional one

Reply:
Done